angularjs-portal-frame/src/main/webapp/portal/notifications/partials/notification-bell.html

```diff
@@ -0,0 +1,7 @@
+<div  ng-controller="NotificationController as notificationIconCtrl" ng-if="$storage.notificationsDemo">
+    <a title="click to view notifications" href="{{notificationIconCtrl.notificationUrl}}">
+        <div  class='notification-badge' ng-class='{ "notification-badge-empty" : notificationIconCtrl.isEmpty }'>
```

Comment:
Fixed by adding `aria-label` with dynamic text based on status

![image](https://cloud.githubusercontent.com/assets/3534544/8701341/d03fa422-2ad6-11e5-9f12-638df9267f99.png)

_Says "you have no notifications at this time"_ on empty state